packages/circuits/circuits/proveReputation.circom

```diff
@@ -15,8 +15,8 @@ include "./incrementalMerkleTree.circom";
 include "./modulo.circom";
 
 template ProveReputation(STATE_TREE_DEPTH, EPOCH_TREE_DEPTH, EPOCH_TREE_ARITY, EPOCH_KEY_NONCE_PER_EPOCH, MAX_REPUTATION_SCORE_BITS) {
-    signal input epoch;
-    signal input nonce;
+    // signal input epoch;
+    // signal input nonce;
```

Comment:
before merge to main, we can delete the comment out lines

Reply:
[3433f43](https://github.com/Unirep/Unirep/pull/216/commits/3433f433f990277ebf199e7561afd5bdfeeb958c)